README.md

```diff
@@ -2,6 +2,8 @@
 
 A project based learning activity for people who are getting started with Git and GitHub.
 
+You can play the game at: https://githubschool.github.io/github-games/
```

Comment:
This update looks nice because the viewer will quickly notice when he opens the file